app/views/groups/show.html.erb

```diff
@@ -31,7 +31,10 @@
           </tr>
         </tbody>
       </table>
-      <% if @viewmodel.group.users_can_apply && !@own_membership %>
+      <% if @viewmodel.leader? %>
+        <%= link_to 'Adatok szerkesztése', edit_group_path(@viewmodel.group), class: 'uk-button uk-button-primary' %>
```

Comment:
I saw you are already using localization  but strangely only in the controllers. Is there a logic you follow with this or just an accident?

Reply:
Shortly, this is only to avoid string literals in controllers.

This project's speciality it will be used only in hungarian, so this doesn't require it. I think it's only a nice to have to display the same texts over different pages, but I think we have a long ways to go before we should do this refactor.

What do you think?